DigitalLearningSolutions.Web.Tests/Controllers/LearningMenu/LearningMenuControllerTests.cs

```diff
@@ -200,6 +204,59 @@ public void Sections_should_StartOrUpdate_course_sessions()
                 .MustNotHaveHappened();
         }
 
+        [Test]
+        public void Tutorials_should_StartOrUpdate_course_sessions()

```

Comment:
Should also have a test checking that it doesn't update the session of the tutorial/section/customisation combo is invalid, as per HEEDLS-225 which I just raised 🙂

Reply:
Yep, I've added tests using a similar pattern to the HEEDLS-225 solution I've PR'ed 🙂